.gitignore

```diff
@@ -58,3 +58,4 @@ build/
 instapk.log*
 ios/Frameworks/
 ios/Runner.xcworkspace/xcshareddata/
+example/ios/Flutter/flutter_export_environment.sh
```

Comment:
i think no need to update git ignore

Reply:
This file contains path information for resources required to project (Path can be very based on computer) which is not required in git.